docs-source/content/faq/configmaps.md

```diff
@@ -0,0 +1,33 @@
+> I need to provide an instance with access to a Config Map
```

Comment:
Both FAQs need a title. The date is also required but the weight is not (unless you want to specify ordering). The FAQ are not "added to the index page" but appear under the category where you locate the MD files.